blueprints/blueprint_tools.py

```diff
@@ -19,19 +19,13 @@
 
 @tools_routes.before_request
 def before_request():
-    geode_functions.create_lock_file(
-        os.path.abspath(flask.current_app.config["LOCK_FOLDER"])
-    )
+    utils_functions.increment_request_counter(flask.current_app)
 
 
 @tools_routes.teardown_request
 def teardown_request(exception):
-    geode_functions.remove_lock_file(
-        os.path.abspath(flask.current_app.config["LOCK_FOLDER"])
-    )
-    geode_functions.create_time_file(
-        os.path.abspath(flask.current_app.config["TIME_FOLDER"])
-    )
+    utils_functions.decrement_request_counter(flask.current_app)
+    utils_functions.update_last_request_time(flask.current_app)
```

Comment:
should we add a teardown fonction in our utils? looks like the same code copy/paste? What happens if we add a third function in the teardown later...

Reply:
You mean a function to wrap this?

Reply:
yes, if we want the same behavior everywhere, we should have a simple de update this everywhere